cmd/cluster-operator/README.md

```diff
@@ -24,17 +23,16 @@ helm install -n cert-manager cert-manager jetstack/cert-manager --set installCRD
 
 ```
 # must run this after cert-manager is ready
-kubectl create namespace kurator-system
-helm install kurator-base out/charts/base-0.1.0.tgz -n kurator-system
-helm install kurator-cluster-operator out/charts/cluster-operator-0.1.0.tgz -n kurator-system
-helm install -f /root/capa-values.yaml -n kurator-system kurator-cluster-operator out/charts/cluster-operator-0.1.0.tgz
+helm install --create-namespace kurator-cluster-operator out/charts/cluster-operator-0.3-dev.tgz -n kurator-system
+
+kubectl logs -l app.kubernetes.io/name=kurator-cluster-operator -n kurator-system --tail=-1
 ```
 
 ## Cleanup
 
 ```
-helm uninstall kurator-cluster-operator -n kurator-system 
-helm uninstall kurator-base -n kurator-system
+helm uninstall kurator-cluster-operator -n kurator-system
 kubectl delete crd $(k get crds | grep cluster.x-k8s.io | awk '{print $1}')
+kubectl delete crd $(k get crds | grep kurator.dev | awk '{print $1}')
```

Comment:
wouldn't they deleted by helm uninstall?

Reply:
this's expected https://helm.sh/docs/topics/charts/#limitations-on-crds